docs/conf.py

```diff
@@ -118,7 +118,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx_rtd_theme'
```

Comment:
If this theme change from `sphinx_rtd_theme` to `default` was in response to build errors: The version of `sphinx_rtd_theme` installed by RTD is ancient and you must force a higher version in `docs.requirements.txt`. Also, as documented in the Zope versions file, `sphinx_rtd_theme` requires `docutils<0.19`. The following entries into `docs/requirements.txt` fixed all these issues for me:

```
docutils<0.19
sphinx_rtd_theme>1
```

Reply:
Done in https://github.com/zopefoundation/BTrees/commit/46b78172219e84d9dfa2c104d05586958b4f3056.